CorsixTH/Lua/world.lua

```diff
@@ -1903,6 +1903,7 @@ function World:newFloatingDollarSign(patient, amount)
   spritelist:setPosition(-17, -60)
   spritelist:setSpeed(0, -1):setLifetime(100)
   spritelist:setSheet(TheApp.gfx:loadSpriteTable("Data", "Money01V"))
+  spritelist:setUseIntermediateBuffer()
```

Comment:
Existing floating dollars likely would need an afterLoad to fix them. But this could be more trouble than it's worth

Reply:
Right, and persist support which I'm not sure if we should do just yet. I'm open to opposing viewpoints, but IMO what we should be doing eventually is drawing TTF text which would negate the need for this bit. I'm not sure how CorsixTH supports loading old saves with persist / depersist and I'm worried about adding temporary baggage on the save file format.

Reply:
Okay then. +1 to just dropping any afterload. The new code will be picked up on any following dollar sign generations.

Reply:
I'm opposed to adding an afterLoad to fix such a slight transient visual artifact.